### DISCUSSION TITLE
I would like to be able to set an HTTP user-agent header when downloading images

### DISCUSSION THREAD

**Discussion by davefxy** [dotnet/maui#11329] [Ideas]
I use a website that requires an HTTP User-Agent header when downloading images. I would like to use image caching but as I said I need to specify a User-Agent header for these system generated requests. It would be fine if Maui provided a User-Agent header of its own. Or allow the app to specify this User-Agent header for system generated HTTP get requests.

**Comment by taublast** [comment]
This could look as being an issue as of 2024. Many sites will not let you download images without it, so MAUI not being able to show an internet image it's rather a bug than a missing feature.

https://foundation.wikimedia.org/wiki/Policy:User-Agent_policy

**Reply by arunadonis** [reply]
Any updates on this issue?
